File: Treinamento/Java/Exercícios_listas/Questão07.java
import java.util.Scanner;
import static java.lang.Math.*;
public class Questão07 {
    public static void main(String[] args) {
        int v_num[] = new int[10];
        int soma = 0;
        Scanner s;
        s = new Scanner(System.in);

        for (int c = 0; c < 10; c++) {
            System.out.print("Digite o " + (c + 1) + "° número: ");
            int n = s.nextInt();
            v_num[c] = n;
        }

        for (int c = 0; c < 10; c++) {
            soma += pow(v_num[c], 2);
        }
        System.out.println("Soma: " + soma);
    }
}

File: Treinamento/C/Exercícios_listas/Questão07.c
#include <stdio.h>

int main() {
    int soma = 0;
    int mult = 1;
    int numeros[5];

    for (int c = 0; c < 5; c++) {
        printf("Digite o %i° número: ", c + 1);
        scanf("%i", &numeros[c]);
        soma += numeros[c];
        mult *= numeros[c];
    }

    printf("Soma: %i\nMultipliação: %i\n", soma, mult);

    for (int c = 0; c < 5; c++) {
        printf("%i ", numeros[c]);
    }
    printf("\n");
}
